canonical-websites/www.ubuntu.com
# 1591 internet of things overview

## Done

* update /internet-of-things overview to vbt
* missing - blockquote accent styles in server/hyperscale

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/internet-of-things)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/7.%20IoT?preview=iot-overview.png) and the [demo](http://www.ubuntu.com-1591-internet-of-things-overview.demo.haus/internet-of-things)

## Issue / Card

Fixes #1591
